packages/edit-site/src/components/sidebar-global-styles-wrapper/index.js

```diff
@@ -131,6 +131,10 @@ export default function GlobalStylesUIWrapper() {
 							// Go to color palettes Global Styles.
 							onPathChange( '/colors/palette' );
 							return;
+						} else if ( blockName === 'typography' ) {
+							// Go to typography Global Styles.
+							onPathChange( '/typography' );
+							return;
 						}
```

Comment:
```suggestion
						}

						if ( blockName === 'typography' ) {
							// Go to typography Global Styles.
							onPathChange( '/typography' );
							return;
						}
```

I think the `else if` is redundant due to the previous early return?

Also it looks like the `/typography` route handling needs to happen in the editor too: https://github.com/WordPress/gutenberg/blob/2a18aae3768da80f94e437c16e8d56e5a9a45e03/packages/edit-site/src/components/global-styles/ui.js#L194-L195

Reply:
Fixed, thanks for spotting that!